controllers/multiclusterobservability/multiclusterobservability_controller.go

```diff
@@ -238,7 +238,9 @@ func (r *MultiClusterObservabilityReconciler) Reconcile(ctx context.Context, req
 	}
 
 	//update status
-	requeueStatusUpdate <- struct{}{}
+	if updateStatusIsRunnning {
+		requeueStatusUpdate <- struct{}{}
+	}
```

Comment:
make sure the status update channel is running before send signal, otherwise the reconcile will hang here.